index.bs

```diff
@@ -2043,9 +2101,16 @@ attributes.
         specified [[#attachment]].
 
     :   <dfn>requireResidentKey</dfn>
-    ::  This member describes the [=[RP]=]'s requirements regarding [=resident credentials=].
-        If the parameter is set to [TRUE], the authenticator MUST create a
-        [=client-side-resident public key credential source=] when creating a [=public key credential=].
+    ::  This member is retained for backwards compatibility but is deprecated in favour of providing *residentKey*.
+        It will be ignored if *residentKey* is supplied and understood by the user-agent. When provided, it describes the 
+        [=[RP]=]'s requirements regarding [=resident credentials=]. If the parameter is set to [TRUE], the authenticator MUST create 
+        a [=client-side-resident public key credential source=] when creating a [=public key credential=].
+
+    :   <dfn>residentKey</dfn>
+    ::  This member supercedes the use of *requireResidentKey* to describes the [=[RP]=]'s requirements regarding 
+        [=resident credentials=]. Only one of *residentKey* or *requireResidentKey* should be present in 
+        {{AuthenticatorSelectionCriteria}}, and use of *residentKey* is encouraged. If both are present 
+        and the user-agent supports it, *residentKey* takes precedence over *requireResidentKey*.
```

Comment:
s/user-agent/[=client=]/

Reply:
Changed both at this location, and in the preceding paragraph.